src/main/resources/messages/messages.properties

```diff
@@ -566,6 +572,10 @@ error.label.not_found=Couldn't find label by the specified title
 error.file.dest_required='dest' parameter required to specify source file path
 error.file.type_required='--type' is required for '--parser-version' option
 
+error.branch.clone=Failed to clone branch
+error.branch.clone_files_based=Clone branch is available only for string-based projects
```

Comment:
Let's make this more generic - `error.string_based_only=This command is only available for string-based projects`